packages/SideNavigation/__tests__/SideNavigation.spec.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import SideNavigation from '../SideNavigation'
+
+describe('SideNavigation', () => {
+  const children = (
+    <SideNavigation verticalSpacing={3}>
+      <SideNavigation.Link href="#">Home</SideNavigation.Link>
+      <SideNavigation.Link href="#one">One</SideNavigation.Link>
+      <SideNavigation.Link href="#two">Two</SideNavigation.Link>
+      <SideNavigation.SubMenu
+        active
+        label="Threefdsfdsfds"
+        id="Three"
+        onClick={() => {}}
+        isOpen={false}
+      >
+        <SideNavigation.Link href="#">Option 1</SideNavigation.Link>
+        <SideNavigation.Link href="#">Option 2</SideNavigation.Link>
+        <SideNavigation.Link href="#">Option 3</SideNavigation.Link>
+      </SideNavigation.SubMenu>
+    </SideNavigation>
+  )
+
+  const doMount = () => mount(children)
+
+  it('renders', () => {
+    const sideNavigation = doMount()
+    expect(sideNavigation).toMatchSnapshot()
+  })
+
+  it('renders without vertical spacing', () => {
+    const childrenNoSpacing = (
+      <SideNavigation>
+        <SideNavigation.Link href="#">Home</SideNavigation.Link>
+        <SideNavigation.SubMenu
+          active
+          label="Threefdsfdsfds"
+          id="Three"
+          onClick={() => {}}
+          isOpen={false}
+        >
+          <SideNavigation.Link href="#">Option 1</SideNavigation.Link>
+        </SideNavigation.SubMenu>
+      </SideNavigation>
+    )
+    const sideNavigationMount = mount(childrenNoSpacing)
+
+    expect(sideNavigationMount).toMatchSnapshot()
+  })
+
+  it('toggles open on submenu', () => {
+    const sideNavigation = doMount()
+
+    sideNavigation
+      .find('SubMenu')
+      .find('button')
+      .simulate('click')
+    expect(sideNavigation.state('open')).toEqual('Three')
+    sideNavigation
+      .find('SubMenu')
+      .find('button')
+      .first()
```

Comment:
Is `first()` needed? `find()` should return the first matching result.

Reply:
It's not. Removed